README.md

```diff
@@ -16,6 +16,7 @@
 <!-- Header block for project -->
 
 [![SLIM](https://img.shields.io/badge/Best%20Practices%20from-SLIM-blue)](https://nasa-ammos.github.io/slim/)
+![DORA Metrics](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/NASA-AMMOS/slim/main/.github/badges/dora-metrics.json)
 
 ![screen-slim](https://github.com/NASA-AMMOS/slim/assets/3129134/d4da5150-aae6-4986-b18e-5c463f8ff38a)
```

Comment:
@yunks128 - can we modify this screen image to show a README with the badge in addition to DevLake?